Dockerfile

```diff
@@ -11,10 +11,9 @@
 ########################################
 # Stage 1: Build the pathfinder binary #
 ########################################
-FROM rust:1.61-alpine AS rust-builder
+FROM rust:1.61 AS rust-builder
```

Comment:
Why does the rust base image change, and final binary change from static to dynamic as in how do we know that `rust:XXX` and `python:YYY-slim` are compatible?

Reply:
The `rust:xxx` are also Debian bullseye-based: https://hub.docker.com/_/rust

And what's the point of keeping the static binary if we know that the base images are compatible?

Reply:
Yeah I was just pointing out that do we know the base images are compatible? Oki now looking at hashes it would seem that python:3.8-slim is bullsey-slim, and similarly for the rust image. Oki, then I propose all of these depend on the bullsey variants directly rather than implicitly?

Reply:
Sure, that's a good idea.